CMakeLists.txt

```diff
@@ -47,21 +47,31 @@ if(NOT CMAKE_BUILD_TYPE)
 endif()
 
 # Version tokens
+set(NVIM_VERSION_MAJOR 0)
+set(NVIM_VERSION_MINOR 1)
+set(NVIM_VERSION_PATCH 0)
+set(NVIM_VERSION_PRERELEASE "-alpha")
+set(NVIM_VERSION_CODENAME  "Whoa!")
```

Comment:
@dnmario I've deleted your message. I don't know if you're a bot, but please stop.